### ISSUE TITLE
Tools - Export - the Author cannot be or start with a number

### ISSUE THREAD

**Issue by GoogleCodeExporter** [viktorthang/wordpress-custom-content-type-manager#294]
```
As the titles says. When I tried to export the CCTM setting it fills my email 
into the Author field (nice). But the problem is that my email starts with 
2046@.... and so the CCTM gives me this:

Please correct the following errors:
The author name cannot exceed 32 characters.

When I changed the field into the plain number "2046" I got the same result.

When I wrote there "me" everything was fine.

*SYSTEM INFO* 
------------------------ 
Plugin Version: 0.9.5.5-pl
WordPress Version: 3.3.1
PHP Version: 5.3.3-7+squeeze3
MySQL Version: 5.1.41-3ubuntu12.7-log
Server OS: Linux
------------------------ 
Other Active plugins: 
 * qTranslate v.2.5.27 [http://www.qianqin.de/qtranslate/]
 * SyntaxHighlighter Evolved v.3.1.3 [http://www.viper007bond.com/wordpress-plugins/syntaxhighlighter/]

```

Original issue reported on code.google.com by `2...@artycok.tv` on 19 Jan 2012 at 1:31

**Comment by GoogleCodeExporter** [comment]
```
the same for CCTM Plugin Version: 0.9.5.6-dev
```

Original comment by `2...@artycok.tv` on 19 Jan 2012 at 1:35

**Comment by GoogleCodeExporter** [comment]
```
Thanks for the report.  This is probably just a validation rule that expects 
the first character to be a letter... investigating.
```

Original comment by `ever...@fireproofsocks.com` on 19 Jan 2012 at 4:59
- Changed state: **InReview**

**Comment by GoogleCodeExporter** [comment]
```
Ah.  Typo.  In includes/CCTM_ImportExport.php line 601:

This:

        elseif ( strlen($raw['author'] > 64) ) {

Should read:
        elseif ( strlen($raw['author']) > 64 ) {


Committed revision 492205.  This will be available in the dev package shortly: 
http://downloads.wordpress.org/plugin/custom-content-type-manager.zip  and it 
will be included in 0.9.5.6
```

Original comment by `ever...@fireproofsocks.com` on 19 Jan 2012 at 5:05
- Changed state: **Fixed**
